src/Package/Impl/DataInspect/VariableViewModel.cs

```diff
@@ -69,7 +67,7 @@ public sealed class VariableViewModel : RSessionDataObject, IIndexedItem {
         }
 
         private static Lazy<VariableViewModel> _ellipsis = Lazy.Create(() => {
-            var instance = new VariableViewModel();
+            var instance = new VariableViewModel(null);
```

Comment:
It has that `public VariableViewModel(ICoreShell coreShell): base(coreShell?.Services) { Index = -1; }`